lib/jnpr/junos/utils/start_shell.py

```diff
@@ -100,18 +112,20 @@ def open(self):
         """
         if self.ON_JUNOS is True:
             self._chan = subprocess.Popen(
-                ["cli", "start", "shell"],
+                ["cli", "start", "shell", self.shell_type],
```

Comment:
What is the behaviour if start shell is None

Reply:
shell_type="csh"  is initialized with csh by default in init of StartShell,